app/workers/subject_import_worker.rb

```diff
@@ -0,0 +1,84 @@
+require 'csv'
+
+class SubjectImportWorker
+  include Sidekiq::Worker
```

Comment:
not yet - adam's setting them up and no normal worker won't pick them up. so up to you about specifying the queue.